std/internal/math/biguintcore.d

```diff
@@ -168,7 +169,7 @@ public:
             return (n & 1) ? cast(uint)(x >> 32) : cast(uint) x;
         }
     }
-public:
+
     ///
```

Comment:
Was this removed because the following method wasn't intended to be public?  Also, does this change somehow relate to this PR?

Reply:
Public is already set a couple of lines above. It's pretty redundant and I don't think it's a problem to include such small things if they are trivial.

Reply:
Fair enough.

Though I'm starting to agree with @andralex that `private` should be attached per declaration rather than wholesale, for better context when reading the code. But that's for another day.